metabase.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Installing Metabase for the staging server
+
+# Eg: ansible-playbook metabase.yml -i hosts -l staging2.ceresfairfood.org.au --ask-become-pass --user <remote-user>
+
+# Certificate was set up manually:
+# certbot certonly --nginx --email "maikel@openfoodnetwork.org.au" -d "metabase.ceresfairfood.org.au" --renew-hook "systemctl reload nginx"
+
+- name: Install Metabase
+  hosts:
+    - vagrant
+    - vagrant-virtualbox
+    - staging
+
+  roles:
+    - role: mysql
+      become: yes
```

Comment:
This is probably how you accidentally reset the Mysql password. It generates a random one on the dev machine. So that password was on my machine when I provisioned the staging server.

Reply:
Yeah, I added a metabase db user in this role and re-ran it. 
Easy fix though, I just copied the original password to my local `mysqlpassword` file, and ran it again. 

It wouldn't have been a problem if I took the time to create a separate password for the new user. I guess that's what happens when you cut corners ;)